contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -273,17 +272,18 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
             return uint256(Error.INSUFFICIENT_SHORTFALL);
         }
 
-        if (liquidationManager.isToxicLiquidation(averageLT, liquidationIncentiveAvg, healthFactor)) {
-            return uint256(Error.TOXIC_LIQUIDATION);
-        }
+        // Call getDynamicLiquidationIncentive from MarketFacet via diamond proxy
+        uint256 dynamicLiquidationIncentive = IMarketFacet(address(this)).getDynamicLiquidationIncentive(
```

Comment:
Would it make sense to move `getDynamicLiquidationIncentive` to the FacetBase?

Reply:
[8b37991](https://github.com/VenusProtocol/venus-protocol/pull/604/commits/8b3799159d7878b56eaf8908baf1295d16b5568e)